extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -176,32 +176,22 @@ private LDAPGuacamoleProperties() {}
     };
 
     /**
-     * Whether or not we should follow referrals.
+     * A time limit on LDAP operations.
      */
-    public static final BooleanGuacamoleProperty LDAP_FOLLOW_REFERRALS = new BooleanGuacamoleProperty() {
-
-        @Override
-        public String getName() { return "ldap-follow-referrals"; }
-
-    };
-
-    /**
-     * Maximum number of referral hops to follow.
-     */
-    public static final IntegerGuacamoleProperty LDAP_MAX_REFERRAL_HOPS = new IntegerGuacamoleProperty() {
+    public static final IntegerGuacamoleProperty LDAP_OPERATION_TIMEOUT = new IntegerGuacamoleProperty() {
 
         @Override
-        public String getName() { return "ldap-max-referral-hops"; }
+        public String getName() { return "ldap-operation-timeout"; }
 
     };
 
     /**
-     * Number of seconds to wait for LDAP operations to complete.
+     * Whether or not to follow referrals.
      */
-    public static final IntegerGuacamoleProperty LDAP_OPERATION_TIMEOUT = new IntegerGuacamoleProperty() {
+    public static final BooleanGuacamoleProperty LDAP_FOLLOW_REFERRALS = new BooleanGuacamoleProperty() {
```

Comment:
I see that `LDAP_FOLLOW_REFERRALS` and `LDAP_OPERATION_TIMEOUT` have switched positions. Any reason for this?

Reply:
I think it was a side-effect of rebasing and the LDAP Referral work that got pushed in ahead of that.

Reply:
Reverted.